images/image_consumer1/data_consumer1.py

```diff
@@ -0,0 +1,56 @@
+from kafka import KafkaConsumer, KafkaProducer
+import json
+import uuid
+import pandas as pd
+import csv
+import time
+from datetime import datetime
+
+server1 = 'broker1:9093'
+server2 = 'broker2:9095'
+server3 = 'broker3:9097'
+
+topic1 = "data_gen1"
+topic2 = "data_gen2"
+
+print(f'{datetime.now()}: config..', flush=True)
+
+consumer1 = KafkaConsumer(topic1,
+                         auto_offset_reset='earliest',
+                         bootstrap_servers=[server1],
```

Comment:
```suggestion
                         bootstrap_servers=[server1, server2, server3],
```
du könntest direkt alle drei Broker angeben. falls broker1 nicht erreichbar ist, kann der Client einen Fallback auf einen anderen machen.

Reply:
stimmt, das wäre sinnvoll